### DISCUSSION TITLE
Zgoda ciasteczkowa - AdGuard

### DISCUSSION THREAD

**Discussion by mlkf2014** [FiltersHeroes/PolishCookieConsent#125] [Pytania i odp] [answered]
Zaczynam na Gicie i proszę wszystkich o wyrozumiałość. Moje pytanie:
Mam na androidzie Agduarda, czy rozszerzenie PZC można zaimportować do rozszerzeń agduarda czy też może to działać jako lista filtrów?

**Comment by admin** [comment, marked as answer]
Podwójne nie.

Teoretycznie w AG może i dałoby się dodać podobne reguły jak w PCZ (trzeba dokonać przeróbki listy https://github.com/PolishFiltersTeam/PolishCookieConsent/blob/master/src/PCB.txt), bo niby tam każdy kod można dodać, ale żeby lista działała trzeba być ważną szychą w firmie AG. Dla zwykłych zarządców list są ograniczenia.

**Comment by mlkf2014** [comment]
Dzięki wielkie za odpowiedź.

**Comment by admin** [comment]
Na Androidzie masz do wyboru przeglądarki ~~Kiwi~~ oraz ~~Yandex~~ i do nich możesz doinstalować PCZ z Chrome Web Store.
Ewentualnie może [Iceraven](https://github.com/fork-maintainers/iceraven-browser) albo Firefox, ale dawno nie sprawdzałem, czy na nich działa.

**Comment by mlkf2014** [comment]
Rozumiem, ale ja używam duck'a a tam nic się nie da zrobić, więc myślałem że może w adguardz spróbuje. Dzięki wielkie.

**Comment by admin** [comment]
Kiedyś był jako `.user.js` ale ten kod już nie jest dostępny raczej online i jest porzucony, jak znajdziesz ostatnią wersje.

[a comment by an automated account is not shown]
